Source/Core/InputCommon/DynamicInputTextureConfiguration.cpp

```diff
@@ -300,24 +300,44 @@ bool DynamicInputTextureConfiguration::GenerateTexture(
     }
   }
 
-  // Load image copy
-  auto base_image = LoadImage(m_base_path + texture_data.m_image_name);
-  bool dirty = false;
+  // Load source texture twice
+  // once as an original image to fallback to if a key or device isn't mapped
+  // the second is the image to write to the textures directory
+  auto original_image = LoadImage(m_base_path + texture_data.m_image_name);
+  auto image_to_write = LoadImage(m_base_path + texture_data.m_image_name);
```

Comment:
`auto image_to_write = original_image;`? I don't think the image needs to be loaded twice -- you just want a copy of the source texture, right?

Reply:
Good catch